### PR TITLE
Add Appium Setup Guide

### PR DESCRIPTION
**Pull request by Aliaksei-Andreyeu**
Adds a document into the user-guides section that describes installation and setup processes for Appium.
Also contains the WebDriverAgent setup section and a F.A.Q.

### CODE REVIEW

**Comment by valfirst** [comment]
Please move images from `ROOT` to `user-guides`

File: docs/modules/user-guides/pages/appium-setup.adoc [hunk 1]
```diff
@@ -0,0 +1,427 @@
...
+
+Follow the instructions below to handle this issue.
+
+==== Manual WDA configuration
+
+NOTE: The next steps are based on https://appium.io/docs/en/drivers/ios-xcuitest-real-devices/#basic-manual-configuration[Appium manual configuration doc]. 
+
+. Add a new Apple Development Certificate for your Apple ID;
++
+.Click to expand the section
```
Comment (valfirst) [inline docs/modules/user-guides/pages/appium-setup.adoc:270]:
```suggestion
.Click to see details
```

File: docs/modules/user-guides/pages/appium-setup.adoc [hunk 2]
```diff
@@ -0,0 +1,427 @@
...
+image:ROOT:appium-add-a-new-certificate.png[]
+====
+. Build WebDriverAgent signed with your newly created certificate;
+.. Open WDA project in Xcode;
++
+TIP: Most common path to the project: `/Applications/Appium.app/Contents/Resources/app/node_modules/appium/node_modules/appium-webdriveragent/WebDriverAgent.xcodeproj`
++
+.. Come up with your unique *Bundle Identifier*. Use it for updating build parameters.
++
+.Click to expand the section
```
Comment (valfirst) [inline docs/modules/user-guides/pages/appium-setup.adoc:294]:
```suggestion
.Click to see details
```

File: docs/modules/user-guides/pages/appium-setup.adoc [hunk 3]
```diff
@@ -0,0 +1,427 @@
...
+*Make sure you've chosen correct device and parameters*
+image:ROOT:appium-check-device-and-parameters.png[]
+
+*Create your unique BundleId and update build parameters*
+image:ROOT:appium-update-build-parameters.png[]
+====
+
+.. Build the project
++
+.Click to expand the section
```
Comment (valfirst) [inline docs/modules/user-guides/pages/appium-setup.adoc:306]:
```suggestion
.Click to see details
```

File: docs/modules/user-guides/pages/appium-setup.adoc [hunk 4]
```diff
@@ -0,0 +1,427 @@
...
+. Tap on "Apple Development: "account name";
+. Choose "Trust";
+
+.Click to view screenshot
+[%collapsible]
+====
+image:ROOT:appium-wda-on-device.png[]
+====
+
+You are ready to run tests! Just add properties from above to your .properties file and use xref:appium-setup.adoc#_running_tests_on_simulators[the same approach as for Simulators]
```
Comment (valfirst) [inline docs/modules/user-guides/pages/appium-setup.adoc:373]:
```suggestion
You are ready to run tests! Just add properties from above to your `*.properties` file and use xref:appium-setup.adoc#_running_tests_on_simulators[the same approach as for Simulators]
```

**Review by valfirst — CHANGES_REQUESTED** [review]

File: docs/modules/user-guides/pages/appium-setup.adoc [hunk 5]
```diff
@@ -0,0 +1,427 @@
...
+- *platformVersion*: the version of the platform to automate
+- *deviceName*: the kind of device to automate
+- *app*: the path to the app you want to automate (but use the browserName capability instead in the case of automating a web browser)
+- *automationName*: the name of the driver you wish to use
+
+There are lots of other capabilities. To find out more please see http://appium.io/docs/en/writing-running-appium/caps/index.html[Appium Capabilities Doc].
+
+=== Setting capabilities in VIVIDUS
+
+Before running vividus scenarios it is necessary to pass Desired Capabilities to Appium Server. Use them as properties with `selenium.grid.capabilities.` prefix. Then just put these capabilities into the appropriate `*.properties` file. Please see examples below:
```
Comment (valfirst) [inline docs/modules/user-guides/pages/appium-setup.adoc:86]:
```suggestion
Before running VIVIDUS scenarios it is necessary to pass Desired Capabilities to Appium Server. Use them as properties with `selenium.grid.capabilities.` prefix. Then just put these capabilities into the appropriate `*.properties` file. Please see examples below:
```

**Review by valfirst — CHANGES_REQUESTED** [review]

**Review by valfirst — APPROVED** [review]

File: docs/modules/user-guides/pages/appium-setup.adoc [hunk 6]
```diff
@@ -0,0 +1,427 @@
...
+selenium.grid.capabilities.automationName=XCUITest
+selenium.grid.capabilities.app=/Users/myDirectory/application.ipa
+
+# Example of Android properties
+selenium.grid.capabilities.platformName=android
+selenium.grid.capabilities.platformVersion=11.0
+selenium.grid.capabilities.deviceName=Pixel 6
+selenium.grid.capabilities.automationName=UIAutomator2
+selenium.grid.capabilities.app=/Users/myDirectory/application.apk
+----
```
Comment (uarlouski) [inline docs/modules/user-guides/pages/appium-setup.adoc:105]: Imho it's better to put each set of props into separate  source section e.g.
.Example of iOS properties
[source, property]
----
......
----

.Example of Android properties
[source, property]
----
......
----

File: docs/modules/user-guides/pages/appium-setup.adoc [hunk 7]
```diff
@@ -0,0 +1,427 @@
...
+selenium.grid.capabilities.appiumVersion=1.20.2
+selenium.grid.url=http://127.0.0.1:4723/wd/hub
+configuration.profiles=mobile_app/android
+configuration.suites=mobile_app/android
+configuration.environments=
+selenium.grid.capabilities.deviceName=Lenovo
+selenium.grid.capabilities.platformName=android
+selenium.grid.capabilities.automationName=UIAutomator2
+----
+====
```
Comment (uarlouski) [inline docs/modules/user-guides/pages/appium-setup.adoc:182]: https://github.com/vividus-framework/vividus/pull/2925/files#r917651417 same here, so end users can copy the content into clipboard without the need to remote unnecessary stuff

File: docs/modules/user-guides/pages/appium-setup.adoc [hunk 8]
```diff
@@ -0,0 +1,427 @@
...
+npm install -g appium
+----
+To run Appium use the command:
+[source,shell]
+----
+appium
+----
+====
+
+== Basic setup
```
Comment (ikalinin1) [inline docs/modules/user-guides/pages/appium-setup.adoc:39]:
```suggestion
== Environment setup
```

```suggestion
== Mobile developer tools setup
```

File: docs/modules/user-guides/pages/appium-setup.adoc [hunk 9]
```diff
@@ -0,0 +1,427 @@
...
+You can get the latest version of Inspector application following the https://github.com/appium/appium-inspector/releases[link].
+
+.Click to view capabilities set up in Inspector
+[%collapsible]
+====
+image::appium-inspector-capabilities-android.png[]    
+image::appium-inspector-capabilities-ios.png[]
+====
+
+== Running tests on Simulators
```
Comment (ikalinin1) [inline docs/modules/user-guides/pages/appium-setup.adoc:125]:
```suggestion
== Running tests on Simulators/Emulators
```

**Review by ikalinin1 — APPROVED** [review]

**Pull request merged**